internal/cluster/transition.go

```diff
@@ -505,6 +495,16 @@ func (th *transitionHandler) PostRefreshCluster(reason string) stateswitch.PostT
 		if err != nil {
 			return err
 		}
+
+		//report cluster install duration metrics in case of error. Cancel and Installed cases are
+		//treated separately in CancelInstallation and CompleteInstallation respectively
+		if sCluster.srcState != swag.StringValue(sCluster.cluster.Status) &&
```

Comment:
Why are we checking statusInfo and not status (error, pending-user-action) ?

Reply:
Fixed